src/azure-cli/HISTORY.rst

```diff
@@ -884,6 +884,10 @@ Release History
 * storage blob generate-sas: User delegation SAS token support with --as-user
 * storage container generate-sas: User delegation SAS token support with --as-user
 
+**RDBMS**
```

Comment:
Please make the release note at the right position, which should be the top of the file.